static/js/features.js

```diff
@@ -22,7 +23,7 @@
 		const result = {};
 		if ( await isEnabled( name ) ) {
 			result.success = false;
-			result.message = `'${ name }' is already enabled.`;
+			result.message = __( `'${ name }' is already enabled.`, 'wp-module-features' );
```

Comment:
These strings will need to be updated to use the `sprintf` method to manage the variables in the strings. Please see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-i18n/#sprintf